mlflow/server/js/src/utils/Utils.js

```diff
@@ -290,7 +290,7 @@ class Utils {
    * @param runTags Object of tag name to MlflowMessages.RunTag instance
    */
   static getRunDisplayName(runTags, runUuid) {
-    return Utils.getRunName(runTags) || "Run " + runUuid;
+    return Utils.getRunName(runTags) || "Run " + runUuid.slice(0, 6);
```

Comment:
Does it make sense to show trailing "..." so this truncation is not confusing.
Thoughts on this: @gioa and @smurching

Reply:
Are the shorten run IDs distinct enough? I am worried they show up the same for different runs. 

Agree with @mparkhe, maybe we should truncate it with text-overflow: ellipsis. I also expect users hover can see the full string, or do a triple-click can copy the whole thing.

Reply:
@mparkhe.
Agree on showing trailing `...`.

@gioa 
> Are the shorten run IDs distinct enough?

I think they are.

> I also expect users hover can see the full string, or do a triple-click can copy the whole thing.

What is the use case of this feature?

Reply:
Doesn't the user need the full run IDs to refer to in the API calls? So they may copy it right there? I might be wrong though. 

Also when I see a trailing ..., I would expect to be able to copy the full string, just a pattern I get used to in other places.

Reply:
How about putting a button on the popover that copies the full run id to the user's clipboard?

Reply:
Example:

![copy-button](https://user-images.githubusercontent.com/17039389/75744595-12ac2c00-5d58-11ea-9d5f-3aa403b462b1.gif)

Reply:
That works too! Might be a little overkill. Could add a run ID in the top info section of a run detail page and support copy there.

Reply:
@Zangr Any thoughts?

Reply:
Tried a copy button on the popover.

![overkill](https://user-images.githubusercontent.com/17039389/75801469-854ff280-5dbe-11ea-961f-85615b9f3857.gif)

Reply:
What do you think? It seems too much going on in this popup. I am okay without it. 

Btw I was talking about adding this in the top info section area. See a mock here:
![MLFlow_RunDetail (1)](https://user-images.githubusercontent.com/1960721/75819508-ae856a00-5d4f-11ea-8ad5-f4b1465db835.png)

Reply:
Yes, it's too much and confusing. The mock looks great!